src/map/trait.h

```diff
@@ -177,13 +177,13 @@ class CTrait
 
 private:
 
-    uint8   m_id;
-    uint8   m_job;
-    uint8   m_level;
-    Mod     m_mod;
-    int16   m_value;
-    uint8   m_rank;
-    uint32  m_meritId;
+    uint8   m_id = { 0 };
```

Comment:
can we use the same style as the example I linked earlier (no equals sign, basically)? just to keep it consistent

Reply:
Yep. pushing in a sec.